x/configuration/client/cli/tx.go

```diff
@@ -138,14 +138,14 @@ func getCmdUpdateConfig(cdc *codec.Codec) *cobra.Command {
 				return err
 			}
 			if validBlockchainID != defaultRegex {
-				config.ValidBlockchainID = validBlockchainID
+				config.ValidResourceURI = validBlockchainID
 			}
 			validBlockchainAddress, err := cmd.Flags().GetString("valid-blockchain-address")
 			if err != nil {
 				return err
 			}
 			if validBlockchainAddress != defaultRegex {
-				config.ValidBlockchainAddress = validBlockchainAddress
+				config.ValidResourceContent = validBlockchainAddress
```

Comment:
`ValidResourceContent` should be `ValidResource`.  (The resource's content could be jpeg data, a calendar invite, etc.)